src/Altinn.Authorization.ServiceDefaults/src/ServiceDefaults/Microsoft.Extensions.Hosting/AltinnServiceDefaultsExtensions.cs

```diff
@@ -402,6 +405,39 @@ private static IConfigurationBuilder AddAltinnDbSecretsJson(this IConfigurationB
     }
 
     private static IConfigurationBuilder AddAltinnKeyVault(this IConfigurationManager manager)
+    {
+        KeyVaultSettings keyVaultSettings = new();
```

Comment:
Most of this method is entirely unneeded. Please use the existing implementation, you only need to change the credentials after all.